### PR TITLE
Changes from AngularDocFX_KR

### PR DESCRIPTION
**Pull request by ESShared**

### CODE REVIEW

File: kr/components/categorychart.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ---
-title: 카테고리 차트 컴포넌트- 네이티브 Angular| Ignite UI for Angular
-_description: Ignite UI for 카테고리 차트 컴포넌트는 데이터 시각화 도메인의 복잡성을 관리 가능한 API로 단순화하여 사용자가 데이터 수집, 그룹 수집, 데이터 속성을 바인딩하고 나머지는 차트 컨트롤이 실행할 수 있도록 합니다.
-_keywords: Ignite UI for Angular, Angular, 네이티브 Angular 컴포넌트 세트, 네이티브 Angular 컨트롤, 네이티브 Angular 컴포넌트, 네이티브 Angular 컴포넌트 라이브러리, Angular 차트, Angular 차트 컨트롤, Angular 차트 예제, Angular 그리드 컴포넌트, Angular 차트 컴포넌트, Angular 카테고리 차트
+title: Category Chart Component - Native Angular | Ignite UI for Angular
+mentionedTypes: ['CategoryChart', 'CategoryChartType']
```
Comment (zdrawku) [inline kr/components/categorychart.md]: @yoshimis do you know from where (in tfs) comes this change. Shouldnt the translation to KR persist after the change? As I see only `mentionedTypes` is added, and the translation to KR is lost

File: kr/components/categorychart_annotation_layers.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
-title: 카테고리 차트 주석 및 오버레이
-_description: Ignite UI for 카테고리 차트 컴포넌트는 데이터 시각화 도메인의 복잡성을 관리 가능한 API로 단순화하여 사용자가 데이터 수집, 그룹 수집, 데이터 속성을 바인딩하고 나머지는 차트 컨트롤이 실행할 수 있도록 합니다.
-_keywords: Ignite UI for Angular, Angular, 네이티브 Angular 컴포넌트 세트, 네이티브 Angular 컨트롤, 네이티브 Angular 컴포넌트, 네이티브 Angular 컴포넌트 라이브러리, Angular 차트, Angular 차트 컨트롤, Angular 차트 예제, Angular 그리드 컴포넌트, Angular 차트 컴포넌트, Angular 카테고리 차트
+title: Annotations
```
Comment (zdrawku) [inline kr/components/categorychart_annotation_layers.md]: Also here?

**Comment by yoshimis** [comment]
@zdrawku Let me confirm with Pam that if we outsource updated contents for KR translation. I do not think we should promote EN and KR mixed contents to production.

File: kr/components/categorychart.md (hunk 1 above)
Comment (valadzhov) [reply kr/components/categorychart.md]: @zdrawku these strings have been updated so we'll have to outsource them to be translated to KR and we'll update them again.

Comment (zdrawku) [reply kr/components/categorychart.md]: I understand

**Pull request merged**